V2/api/evmPrehooktoEVM/src/index.ts

```diff
@@ -125,16 +125,14 @@ const updateTransactionStatus = async (txHash: string, requestId: string) => {
     toChain: toChainId,
     toToken: toToken,
     toAddress: signer.address,
-    slippageConfig: {
-      autoMode: 1,
-    },
+    slippage: 1,  //optional, Squid will dynamically calculate if removed
     preHook: {
       chainType: "evm",
       fundAmount: amount.toString(),
       fundToken: "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE", // Native ETH
       provider: "Integration Test",

```

Comment:
Could you put a comment next to the provider `// This should be the name of your project, or application which is triggering the post hook`

Reply:
Yes, adding!